<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnCantidad</class>
 <widget class="QDialog" name="vtnCantidad">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>162</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>211</width>
    <height>162</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>211</width>
    <height>162</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Cantidad</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../imagenes/info.png</normaloff>../imagenes/info.png</iconset>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:century gothic;
font-size: 15px;
}
#vtnCantidad{
 background:#fefefe;
}
QPushButton{
border-radius:15px;
background:#ffc001;
}

QPushButton:hover{
background:#dea806;
}
QLineEdit{
	border:none;
	border-bottom: 1px solid black;
	background:transparent;
}

</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;¿Cuánto Desea?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptardialogSalida">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineCantidaddialogSalida">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelardialogSalida">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LineCantidaddialogSalida</tabstop>
  <tabstop>btnAceptardialogSalida</tabstop>
  <tabstop>btnCancelardialogSalida</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
